atlasdb-config/src/main/java/com/palantir/atlasdb/config/TimeLockRequestBatcherProviders.java

```diff
@@ -17,9 +17,12 @@
 package com.palantir.atlasdb.config;
 
 import com.palantir.lock.client.LeaderTimeCoalescingBatcher;
+import com.palantir.lock.client.MultiClientTransactionStarter;
 import org.immutables.value.Value;
 
 @Value.Immutable
 public interface TimeLockRequestBatcherProviders {
     TimeLockRequestBatcherProvider<LeaderTimeCoalescingBatcher> leaderTimeBatcherProvider();
+
+    TimeLockRequestBatcherProvider<MultiClientTransactionStarter> startTransactionsBatcherProvider();
```

Comment:
nit: given the class is already `...BatcherProviders`, I'd just name this `leaderTime()` or `startTransactions()` respectively.